src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/controller/isl/IslFsm.java

```diff
@@ -486,11 +525,9 @@ private void fireBecomeStateEvent(IslFsmContext context) {
         fire(route, context);
     }
 
-    private boolean shouldSetupBfd() {
+    private boolean canSetupBfd() {
         // TODO(surabujin): ensure the switch is BFD capable
-
-        return isPerIslBfdToggleEnabled(reference.getSource(), reference.getDest())
-                || isPerIslBfdToggleEnabled(reference.getDest(), reference.getSource());
+        return true;
```

Comment:
Do we still need this method?

Reply:
I would like to keep it and follow TODO placed there i.e. "ensure the switch is BFD capable".